geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/geotools/SimpleFeatureOrdering.scala

```diff
@@ -10,64 +10,66 @@ package org.locationtech.geomesa.utils.geotools
 
 import org.locationtech.geomesa.utils.collection.TieredOrdering
 import org.opengis.feature.simple.{SimpleFeature, SimpleFeatureType}
-import org.opengis.filter.expression.PropertyName
 import org.opengis.filter.sort.{SortBy, SortOrder}
 
 import scala.math.Ordering
 
 /**
-  * Ordering for simple features. Assumes that any attributes implement `Comparable`
+  * Ordering for simple features
```

Comment:
This change probably lines up perfectly with the change above between Comparable[Any] and AnyRef, right?

Reply:
yeah, i switched to using an ordering in DeltaWriter so that we could compare non-comparable lists, and implemented the ordering here.